src/casync-http.c

```diff
@@ -315,7 +315,7 @@ static int acquire_file(CaRemote *rr,
         }
 
         if (arg_verbose)
-                log_info("Acquiring %s...", url);
+                log_debug("Acquiring %s...", url);
```

Comment:
I only changed the log level, but I also wondered if we should remove `arg_verbose` here. And more generally, how is verbose supposed to play along log levels?

Reply:
yeah, log_debug stuff should be unaffected by arg_verbose. Hence, it's fine to downgrade this, but then also please drop the conditionalization by arg_verbose

Reply:
Done